setup.cfg

```diff
@@ -22,6 +22,8 @@ classifiers =
     Programming Language :: Python :: 3.6
```

Comment:
Can be removed IMO since it's end of life.

Reply:
Done